posts/2020-09-30-microprofile-language-support-liberty.adoc

```diff
@@ -0,0 +1,155 @@
+
+== Installing the extensions
+
+The following extensions are available on the VS Code Marketplace to support developing your Open Liberty applications with MicroProfile.
+
+- https://marketplace.visualstudio.com/items?itemName=redhat.vscode-microprofile[Tools for MicroProfile]: Language support for MicroProfile APIs and properties via the https://github.com/eclipse/lsp4mp[Language Server for Eclipse].
+
+- https://marketplace.visualstudio.com/items?itemName=Open-Liberty.liberty-dev-vscode-ext[Open Liberty Tools]: Support for development with Open Liberty.
+
+- https://marketplace.visualstudio.com/items?itemName=MicroProfile-Community.mp-starter-vscode-ext[MicroProfile Starter]: Quickly generate a MicroProfile Maven project.
+
+== Setting up the project
+
+=== Create a new MicroProfile project
+
+Generate a new Open Liberty Maven application with the MicroProfile Starter.
+
+[.img_border_light]
+image::/img/blog/mp-starter-vscode-marketplace.png[MicroProfile Starter for VS Code]
+
+=== Configure an existing project
+
+Modify an existing Maven or Gradle project for use with Open Liberty and MicroProfile.
+
+For Maven projects, add the Liberty Maven Plugin to your `pom.xml` file.
+
+[source,xml]
+----
+<plugin>
+    <groupId>io.openliberty.tools</groupId>
+    <artifactId>liberty-maven-plugin</artifactId>
+    <version>3.3-M3</version>
+</plugin>
+----
+
+Add the MicroProfile dependency to your `pom.xml` file.
+
+[source, xml]
+----
+<dependency>
+    <groupId>org.eclipse.microprofile</groupId>
+    <artifactId>microprofile</artifactId>
+    <version>3.3</version>
+    <type>pom</type>
+    <scope>provided</scope>
+</dependency>
+----
+
+For Gradle projects, add the Liberty Gradle Plugin to your `build.gradle` file.
+
+[source,groovy]
+----
+apply plugin: 'liberty'
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.1-M2'
+    }
+}
+----
+
+Add the MicroProfile dependency to your `build.gradle` file.
+
+[source,groovy]
+----
+dependencies {
+    providedCompile 'org.eclipse.microprofile:microprofile:3.2'
+}
+----
+
+Add the MicroProfile features that you would like to use in your Open Liberty `server.xml` file.
+
+[source, xml]
+----
+<featureManager>
+    <feature>jaxrs-2.1</feature>
+    <feature>jsonp-1.1</feature>
+    <feature>cdi-2.0</feature>
+    <feature>mpMetrics-2.0</feature>
+    <feature>mpConfig-1.3</feature>
+    <feature>mpHealth-2.2</feature>
+</featureManager>
+----
+
+== Starting dev mode
+
+Start your Open Liberty application in dev mode using the Liberty Dev Dashboard, or via command line using `mvn liberty:dev`/`gradle libertyDev`. Dev mode provides hot reload and deployment, so your code changes will automatically be reflected in your running application.
```

Comment:
Should we mention liberty dev dashboard is part of vscode? maybe something like:  

Start your Open Liberty application in dev mode using vscode with the Liberty Dev Dashboard, or via command line using?